etl/steps/data/garden/animal_welfare/2023-08-14/number_of_farmed_fish.meta.yml

```diff
@@ -0,0 +1,36 @@
+definitions:
+  common: |
+    This study estimates the annual number of farmed fish slaughtered for food using data on aquaculture production tonnages (from the Food and Agriculture Organization of the United Nations) and mean weights estimated from available fish size data (from a range of internet sources).
+
+    Note that this estimate does not include:
+    - Non fish species, e.g. lobsters.
+    - Fish deaths arising in fish farms prior to harvest, in the capture of wild fish for feed or re-stocking, or from environmental impacts on surrounding wildlife.
+    - Fish farmed for use as bait, whose small production tonnages may comprise large numbers of fish due to their very small size of just a few grams.
+    - Species without estimated mean weight (which could be an additional 17% of total fish production).
```

Comment:
I also don't see this filtered out. Please let me know if I'm wrong

Reply:
Species without estimated mean weight, by construction, will not have an estimate on the number of individual fish, as far as I understand. At the top of the spreadsheet they take the sum of all countries' numbers of fish, and add a 17% to account for species without estimated mean weight. That's why I mention it here. But I'm not sure about this, and the documentation is a bit spread around the web site and some papers.